solutions/coreclr/test-runner.sh

```diff
@@ -2,7 +2,7 @@
 
 USAGE="$0 path-to-myst exec|exec-sgx|exec-linux ext2|cpio <test_list_file>"
 NPROCS=$(grep -c ^processor /proc/cpuinfo)
-#NPROCS=1
+NPROCS=2
```

Comment:
Any reason to reduce the parallelism?

Reply:
Seems reduce the parallelism doesn't improve the speed. Will revert back.